Utils/UtilsServer.go

```diff
@@ -51,13 +51,15 @@ This function does not return until the communicator is stopped, or returns in c
 
 – Returns:
   - bool – true if the communicator was started or was already started, false if it an error occurred and it did not
-	start
+    start
 */
 func StartCommunicatorSERVER() bool {
 	if srvComm_started_GL {
 		return true
 	}
 	srvComm_started_GL = true
+	log.Println("Communicator start")
+	log.Println("srvComm_connected_GL:", srvComm_connected_GL)
```

Comment:
Can you remove these 2 lines? Not because they bother me at all, but just because there's already a log to the communicator start below on line 176 xD. And also because `srvComm_connected_GL` is always false on line 62. It only becomes true after the log on line 176 comes, and becomes false after the log on line 188 comes.

Reply:
lol oh ok